cedar-example-use-cases/github_example/README.md

```diff
@@ -171,3 +186,24 @@ permit (
 )
 when { principal in resource.admins };
 ```
+
+## Tests
+
+We use the following entities for our tests, included in the `entities.json` file:
+* There are 3 `User` entities, `User::"alice"`, `User::"bob"`, `User::"jane"`.
+  *  Alice is a member of the `common_knowledge_writers` and `uncommon_knowledge_writers` user groups.
+  *  Bob is a member of the `tiny_corp_owners` organization.
+  *  Jane is a member of the `common_knowledge_maintainers` user group and the `team_that_can_read_everything` team.
+* There are 3 `Repository` entities, `common_knowledge`, `uncommon_knowledge`, and `secret`.
+* There are 3 `UserGroup` entities, `common_knowledge_writers`, `uncommon_knowledge_writers`, and `common_knowledge_maintainers` that are children of the `common_knowledge` and `uncommon_knowledge` repositories, respectively.
+* The 1 `Organization` entity, `tiny_corp_owners`, has admin priveleges to all 3 repositories.
+* The 1 `Team` entity, `team_that_can_read_everything`, has read priveleges to all 3 repositories.
+
+Here are some authz requests to test, included in the `ALLOW` and `DENY` folders:
+* Alice tries to pull common_knowledge: ALLOW because she has read permissions through the `common_knowledge_writers` user group.
```

Comment:
Since you're using the name with an underscore, I'd use backticks - so either
```suggestion
* Alice tries to pull common knowledge: ALLOW because she has read permissions through the `common_knowledge_writers` user group.
```
or
```suggestion
* Alice tries to pull `common_knowledge`: ALLOW because she has read permissions through the `common_knowledge_writers` user group.
```

Same comments goes for the next two cases (with `uncommon_knowledge`).